### ISSUE TITLE
JAWS (Screen reader) doesn't announce the group label information when the focus moves to the associated list items under 'View dropdown’.

### ISSUE THREAD

**Issue by sofisa16** [ng-select/ng-select#2397]
### Describe the bug

Reproducible at https://stackblitz.com/edit/ng-select-custom-template-t1sfr1?file=app%2Fapp.component.ts

Right now, JAWS does not announce the group labels United States and Argentina. It only pronounce the contents inside does groups. It is expected that the group labels are pronounced that Windows Narrator does.

![image](https://github.com/ng-select/ng-select/assets/14502698/b2208137-bd07-4a64-9fe5-99b61daacb40)

### Steps to Reproduce

Go to <Grouping> header and start using the select box while using JAWS too.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://stackblitz.com/edit/ng-select-custom-template-t1sfr1?file=app%2Fapp.component.ts

### Expected behaviour

It is expected that the group labels are pronounced that Windows Narrator does.

### What version of the library you see this issue?

angular

### What version of Angular are you using?

Angular 6

### Screenshots

![image](https://github.com/ng-select/ng-select/assets/14502698/2455db65-6261-463f-9e51-99d43cddbeb2)
